salt/metalk8s/kubernetes/apiserver/installed.sls

```diff
@@ -80,15 +80,13 @@ Set up default basic auth htpasswd:
 {%- set host = grains['metalk8s']['control_plane_ip'] %}
 
 Create kube-apiserver Pod manifest:
-  file.managed:
+  metalk8s.static_pod_managed:
     - name: /etc/kubernetes/manifests/kube-apiserver.yaml
     - source: salt://metalk8s/kubernetes/files/control-plane-manifest.yaml
-    - template: jinja
-    - user: root
-    - group: root
-    - mode: 644
-    - makedirs: True
-    - dir_mode: 750
+    - config_files:
+        - {{ htpasswd_path }}
+        - /etc/keepalived/check-apiserver.sh
```

Comment:
These only when `keepalived` is enabled, right?

Reply:
Done